### ISSUE TITLE
Problem with the size of x360ce window

### ISSUE THREAD

**Issue by ssas39409** [x360ce/x360ce#1433]
Hello my problem is that no matter what I do x360ce will open like this. 
![image](https://user-images.githubusercontent.com/131950821/234787071-4c095cda-781f-43ab-8d53-930c38d418b6.png)
And if I stretch it it gets like this 
![image](https://user-images.githubusercontent.com/131950821/234787196-8c84e766-cc34-4559-add7-6233bc0c53a0.png)
It works well in all my other PC's but here don't and I don't know why it could be

**Comment by kotenok2000** [comment]
Right-click the application, select Properties, select the Compatibility tab, and then select the Disable display scaling on high DPI settings check box.
Or set screen scaling to 100% systemwide at System>Display>Scale

**Comment by cmwartime** [comment]
I don't have 'Disable display scaling on high DPI settings'.
Available Settings Choices:
Disable fullscreen optimizations
Run as administrator (chosen)
Register this program for restart

Button: Change high DPI settings
I chose High DPI scaling override

Still not fixed.  Any suggestions?

**Comment by CornerstoneN** [comment]
Did you ever solve it? Ive run into the same problem

**Comment by randomRedMage** [comment]
### Solution
@kotenok2000 provided the solution, it was just a bit hard to understand and may have been based on a different build of Windows.  After experimenting, I found the steps that worked for my build of Windows 11.

I hope this helps.

### Windows Build
Edition	Windows 11 Pro
Version	24H2
OS build	26100.6899
Experience	Windows Feature Experience Pack 1000.26100.253.0

### Steps to fix:
1. Right click on x360ce.exe.
2. Select Properties.
3. Select the Compatibility tab.
4. Near the bottom of the modal in the "Settings" section, select "Change high DPI settings"
5. In the High DPI settings for x360ce.exe modal check the box for "Override high DPI scaling behavior." in the high DPI scaling behavior section.
6. Set the Scaling performed by dropdown to "System" or "System(Enhanced)".
7. Select OK then OK to close out the modals.
8. Open the application.

### Video demonstration:
https://github.com/user-attachments/assets/a14422d9-6acd-4cc3-8cea-6edba3eb5709

**Comment by lowriderdog372** [comment]
> ### Solution
> [@kotenok2000](https://github.com/kotenok2000) provided the solution, it was just a bit hard to understand and may have been based on a different build of Windows. After experimenting, I found the steps that worked for my build of Windows 11.
> 
> I hope this helps.

It did.  Thank you for your documentation.

**Comment by EvoJocys** [comment]
Fixed. From the changelog:

4.17.22 — Fixed: High-DPI rendering - whole UI scales with monitor zoom; also fixes intermittent empty window title.

Download: https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
